src/encoded/loadxl.py

```diff
@@ -692,6 +693,9 @@ def get_pipeline(testapp, docsdir, test_only, item_type, phase=None, method=None
     'reference_epigenome': [
         remove_keys('related_datasets', 'supersedes'),
     ],
+    'disease_series': [
+        remove_keys('related_datasets', 'supersedes'),
```

Comment:
Reverse link and superseded_by weren't defined in the types file for this series. Do we anticipate needing it?